lib/bsdlib/bsdlib.ld

```diff
@@ -0,0 +1,3 @@
+#include <bsd_platform.h>
+__bsdlib = BSD_RESERVED_MEMORY_ADDRESS;
```

Comment:
I can't find where this is used.

Reply:
If you want to place .bsdlib at BSD_RESERVED_MEMORY_ADDRESS, do

    . = BSD_RESERVED_MEMORY_ADDRESS;

Reply:
Is there a difference?

Reply:
The version 62c7147 at least creates the section at the correct address. I'm a bit unsure about the implication of having it that way. Also, it seems like the `static u8_t __attribute__((section (".bsdlib"), unused)) bsd_mem[BSD_RESERVED_MEMORY_SIZE];
` is picked up by something other than the section.